.travis.yml

```diff
@@ -8,5 +8,5 @@ script:
 
 env:
     matrix:
-        - DOCKERFILE_LOCATION='.'
-        - DOCKERFILE_LOCATION='./alpine'
+        - DOCKERFILE_LOCATION='./2020.10/buster'
+        - DOCKERFILE_LOCATION='./2020.10/alpine3.12'
```

Comment:
The problem I see with this is that it's too version-locked. It would probably be better to have two environment variables, one with the version, the other with the directory, so that we could build any combination of them without having to list every singe version we use from now on.

Reply:
Is it like the following?
```yaml
script:
    - docker build -t rakudo-star ./$RAKUDO_STAR_VERSION/$BASE_IMAGE

env:
    matrix:
        - RAKUDO_STAR_VERSION='2020.10'
        - BASE_IMAGE='buster'
        - BASE_IMAGE='alpine3.12'
```